src/main/resources/license-data.json

```diff
@@ -2241,6 +2241,58 @@
             "additionalLicenses": "CC0-1.0",
             "licenseSource": "data"
         },
+        "com.zaxxer:HikariCP": {
+            "license": "Apache-2.0",
+            "licenseSource": "data"
+        },
+        "com.zaxxer:HikariCP-java6": {
+            "license": "Apache-2.0",
+            "licenseSource": "data"
+        },
+        "com.typesafe.play:anorm-tokenizer_2.11": {
```

Comment:
@ilx I wonder if Hikari and anorm were added for the `akka-es` module and are not needed in `master` as dependencies.